runtime/ignite/iginite.go

```diff
@@ -0,0 +1,404 @@
+package ignite
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/srl-labs/containerlab/runtime"
+	"github.com/srl-labs/containerlab/types"
+	"github.com/srl-labs/containerlab/utils"
+	api "github.com/weaveworks/ignite/pkg/apis/ignite"
+	meta "github.com/weaveworks/ignite/pkg/apis/meta/v1alpha1"
+	igniteConstants "github.com/weaveworks/ignite/pkg/constants"
+	"github.com/weaveworks/ignite/pkg/dmlegacy"
+	"github.com/weaveworks/ignite/pkg/filter"
+	"github.com/weaveworks/ignite/pkg/metadata"
+	"github.com/weaveworks/ignite/pkg/operations"
+	"github.com/weaveworks/ignite/pkg/providers"
+	igniteDocker "github.com/weaveworks/ignite/pkg/providers/docker"
+	"github.com/weaveworks/ignite/pkg/providers/ignite"
+	"github.com/weaveworks/ignite/pkg/util"
+)
+
+const (
+	RuntimeName      = "ignite"
+	dockerRuntime    = "docker"
+	defaultTimeout   = 30 * time.Second
+	kvmPath          = "/dev/kvm"
+	dockerBridge     = "docker-bridge"
```

Comment:
minor issue, but probably worth renaming `defaultNetworkPluginName`. I thought it was an actual bridge name

Reply:
yep, makes sense